startup/Gaffer/gafferCortexCompatibility.py

```diff
@@ -35,11 +35,18 @@
 ##########################################################################
 
 import Gaffer
-import GafferCortex
 
-# Backwards compatibility - import classes from GafferCortex into
-# the Gaffer namespace.
-for name in dir( GafferCortex ) :
-	if name.endswith( "__" ) :
-		continue
-	setattr( Gaffer, name, getattr( GafferCortex, name ) )
+try :
+
+	import GafferCortex
+
+	# Backwards compatibility - import classes from GafferCortex into
+	# the Gaffer namespace.
+	for name in dir( GafferCortex ) :
```

Comment:
Oof. I'm sure you're trying to be as careful as possible here, but I really hope we're not relying on this ancient config files any longer...